src/SamplesApp/UITests.Shared/Utilities/ControlStateViewer.xaml.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using System.Reflection;
+
+namespace MUXControlsTestApp.Utilities
+{
+	// Test control intended to help you see all the different visual states your control can be in
+	public sealed partial class ControlStateViewer : UserControl
+	{
+		Type _controlType;
+		List<string> _states;
+
+		public ControlStateViewer()
+		{
+			this.InitializeComponent();
+		}
+
+		public Type ControlType
+		{
+			get
+			{
+				return _controlType;
+			}
+
+			set
+			{
+				_controlType = value;
+				UpdateGrid();
+			}
+		}
+
+		public List<string> States
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this property to use a generic collection designed for inheritance.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5246693)